docusign-connector/config/rest-clients.yaml

```diff
@@ -8,12 +8,14 @@ RestClients:
     - ch.ivyteam.ivy.rest.client.mapper.JsonFeature
     - com.axonivy.connector.docusign.auth.OAuth2Feature
     Properties:
-      AUTH.integrationKey: your-app-key
-      AUTH.secretKey: ${decrypt:}
-      AUTH.systemUserId: ''
-      AUTH.systemKeyFile: docusign.pem
-      AUTH.scope: signature impersonation
-      AUTH.baseUri: https://account-d.docusign.com/oauth
+      AUTH.integrationKey: ${ivy.var.docusign-connector.integrationKey}
+      AUTH.secretKey: ${ivy.var.docusign-connector.secretKey}
```

Comment:
this is a secret and should be decrypted: unfortunately there is not much tooling to do it but a workaround:
- add a password -property to the rest-client using the rest-client-editor
- fill the variable name ${ivy.var.xxx} as value

Reply:
ok